### DISCUSSION TITLE
arvrcp only question

### DISCUSSION THREAD

**Discussion by golfdubcrazy** [pschatzmann/ESP32-A2DP#106] [Q&A] [answered]
Hi great code big help,

i am trying to build a bike handle bar controller to control a phone connected with headphones.
i know you have to enable a2dp to enable avrcp, but i only need the avrcp. is there a way of the esp32 rejecting incoming audio and forcing it to stay on the phone ? 
also will be looking into the HFP (hand free profile) to have a call answer reject button.

Any help given would be much appreciated. 

Thanks
M

**Comment by pschatzmann** [comment, marked as answer]
I am sorry but I haven't used AVRCP or HEP. So this is not the right place to ask this question. 
I suggest that you try https://www.esp32.com/viewforum.php?f=23
